crates/uv/tests/lock.rs

```diff
@@ -5834,7 +5834,13 @@ fn lock_dev_transitive() -> Result<()> {
 /// Avoid persisting registry credentials in `uv.lock`.
 #[test]
 fn lock_redact_https() -> Result<()> {
-    let context = TestContext::new("3.12");
+    // This test in particular seems to prompt a link mode warning
+    // that occurs when hardlinking fails. In particular, in this test,
+    // uv tries to hardlink between `/tmp` and `~/.local`, which on my
+    // system fails because `/tmp` is a different file system (a ramdisk).
```

Comment:
That's weird. What are we placing in `/tmp`? We should be avoiding most use of `/tmp` in tests now. I don't see anything particular about this test.